metecho/api/jobs.py

```diff
@@ -56,6 +56,8 @@ def creating_gh_branch(instance):
         yield
     finally:
         instance.currently_creating_branch = False
+        instance.save()
+        instance.notify_changed(originating_user_id=None)
```

Comment:
@jerivas This is tangential to this PR, but I noticed that an error while initially creating a new Epic GH branch (e.g. for one of our sample projects that isn't a real SF package, like `Books`) would lead to a stuck state of `currently_creating_branch = True`.